scan/cli.go

```diff
@@ -33,8 +36,21 @@ import (
 
 const auditScanCategory = "Audit & Scan"
 
+const curationAuditCategory = "Curation Audit"
```

Comment:
Let's remove this const and use `auditScanCategory` for the `curation-audit` command category.